content/installation/setup/auth/Configure-http-header.md

```diff
@@ -0,0 +1,13 @@
+<!--
+title: "Configure HTTP Header"
+description: "Configure Contrast to connect to HTTP Header"
+tags: "installation setup EOP system settings connect http header "
+-->
+Contrast supports certificate login through a proxy using a HTTP Header token. To enable HTTP Header login: 
+
+* Update the property file by changing the ```authentication.mode``` in *~/path_to_contrast_installation/conf/auth.properties* to ```http_header```. 
```

Comment:
You only need one backtick for inline code formatting. Triple backtick is for [fenced code blocks](https://help.github.com/articles/creating-and-highlighting-code-blocks/#fenced-code-blocks).